brutils/pis.py

```diff
@@ -105,7 +78,12 @@ def is_valid(pis: str) -> bool:
     True
 
     """
-    return isinstance(pis, str) and validate(pis)
+
+    if isinstance(pis, str) and len(pis) == 11 and pis.isdigit():
+        if pis[-1] == str(_checksum(pis[:-1])):
+            return True
+
+    return False
```

Comment:
Tem uma dica que é, se tem um if dentro do outro if direto, sempre dá para colocar tudo em um if só com um "and"

```
    return isinstance(pis, str) and len(pis) == 11 and pis.isdigit() and pis[-1] == str(_checksum(pis[:-1]))
```

ou também dá como tinha tentando sugerir antes, mas que tinha esquecido dos not:

```
    if not isinstance(pis, str) or len(pis) != 11 or not pis.isdigit():
        return False

    return  pis[-1] == str(_checksum(pis[:-1]))
```

Como vc preferir :) A gente pode fazer um pair com isso para treinar mais as diferentes opções. Do jeito que tá agora funciona, só não estaria muito "pythônico"